programs/mpl-core/src/state/collection.rs

```diff
@@ -305,11 +305,13 @@ impl CollectionV1 {
     }
 
     /// Increment size of the Collection
-    pub fn increment(&mut self) -> Result<(), ProgramError> {
-        self.num_minted = self
-            .num_minted
-            .checked_add(1)
-            .ok_or(MplCoreError::NumericalOverflowError)?;
+    pub fn increment(&mut self, mint: bool) -> Result<(), ProgramError> {
```

Comment:
nit: prefer using more understandable method names by either splitting the functions to do separate things like `increment_size` and `increment_minted` or instead compose the functions like `increment` and `increment_with_minted` which calls `increment` internally.

`increment(true)` will kind of force me to inspect the implementation to remember what exactly it was trying to do

Reply:
Updated methods to be `increment_size`, `increment_minted`, and `decrement_size`